src/inc/LibraryIncludes.h

```diff
@@ -79,7 +79,9 @@
 #include <wrl.h>
 
 // TIL - Terminal Implementation Library
+#ifndef BLOCK_TIL // Certain projects may want to include TIL manually to gain superpowers

```

Comment:
curiosity: why would someone want this?

Reply:
There's a catch-22 between including TIL in the precomp and also including it late enough in your header include process for it to light up things like D2D color conversions. This is intended to let the minority-case of projects (like DXEngine) block its inclusion and include it later manually so the D2D conversions can light up.

Reply:
okay, that makes sense